src/universal/modules/teamDashboard/components/ProjectColumn/ProjectColumn.js

```diff
@@ -167,6 +167,9 @@ class ProjectColumn extends Component {
         </div>
         <div className={css(styles.columnBody)}>
           <div className={css(styles.columnInner)}>
+            {(projects.length === 0 && label === 'done') &&
```

Comment:
Of the above, I like the "only do this for the first 5 meetings." In a future onboarding epic, we can change this to be something like a link that says "don't show me this again"